apps/meteor/app/2fa/server/methods/checkCodesRemaining.ts

```diff
@@ -1,7 +1,7 @@
-import type { ServerMethods } from '@rocket.chat/ui-contexts';
+import type { ServerMethods } from '@rocket.chat/ddp-client';
```

Comment:
ddp-client is an EE package; let's avoid adding a ton of new references to it from non-ee code.

Reply:
dismissing my previous review as we'll be moving the ddp-client package out of the ee folder.